src/main/java/bio/terra/service/resourcemanagement/azure/AzureStorageAccountService.java

```diff
@@ -198,21 +198,25 @@ public AzureStorageAccountResource retrieveStorageAccountById(UUID storageAccoun
     return resourceDao.retrieveStorageAccountById(storageAccountId);
   }
 
-  /**
-   * Delete metadata and cloud storage account resource. Note: this runs in a transaction so if the
-   * cloud delete fails, the metadata deleted should rollback
-   */
-  @Transactional(propagation = Propagation.REQUIRED, isolation = Isolation.SERIALIZABLE)
-  public void deleteCloudStorageAccount(
+  public void deleteCloudStorageAccountAndMetadata(
       AzureStorageAccountResource storageAccountResource, String flightId) {
+    deleteCloudStorageAccount(storageAccountResource);
+    deleteCloudStorageAccountMetadata(storageAccountResource.getName(), flightId);
```

Comment:
Not sure it's worth having a method instead of having the caller make these two calls directly. It looks like it's only used once in this PR.

Reply:
True - removed!